recipes/anaquin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# download and copy required dependencies
+# klib
+wget https://github.com/attractivechaos/klib/archive/spawn-final.tar.gz
```

Comment:
If you can it would be great to put this into a dependency, or at least get a fixed version of it.